src/rdkafka_request.c

```diff
@@ -2187,26 +2190,39 @@ static void rd_kafka_handle_Metadata(rd_kafka_t *rk,
  * @param allow_auto_create_topics - allow broker-side auto topic creation.
  *                                   This is best-effort, depending on broker
  *                                   config and version.
+ * @param include_cluster_authorized_operations - request for cluster
+ *                      authorized operations.
+ * @param include_topic_authorized_operations - request for topic authorized
+ *                      operations.
  * @param cgrp_update - Update cgrp in parse_Metadata (see comment there).
  * @param force_racks - Force partition to rack mapping computation in
  *                      parse_Metadata (see comment there).
  * @param rko       - (optional) rko with replyq for handling response.
  *                    Specifying an rko forces a metadata request even if
  *                    there is already a matching one in-transit.
- *
+ * @param resp_cb - callback to be used for handling response. If NULL, then
+ *                     rd_kafka_handle_Metadata() is used.
+ * @param force - 1: force a full request.
```

Comment:
Can we explain more? like full request including all the topics and broker information? What happens if there is already a full request going on.